src/core.c/Int.pm6

```diff
@@ -381,23 +381,21 @@ multi sub infix:<%%>(int $a, int $b --> Bool:D) {
     nqp::hllbool(nqp::iseq_i(nqp::mod_i($a, $b), 0))
 }
 
-my constant UINT64_UPPER = nqp::pow_I(2, 64, Num, Int);
-
 multi sub infix:<**>(Int:D $a, Int:D $b --> Real:D) {
     my $power;
     if nqp::isge_I($b, 0) {
```

Comment:
Have you tried to make this a single ternary instead of an `if` / `else`?   It probably doesn't matter for inlining, as it becomes too large after inlining `CREATE_RATIONAL_FROM_INTS`

Reply:
I'm currently experimenting with some of the other Rakudo-only changes I tried in https://github.com/rakudo/rakudo/pull/4320 since that was before new-disp, but so far I haven't seen anything worthwhile.